### PR TITLE
Stand-o-rama - rebalances a bunch of holoparasite stuff

### PR DESCRIPTION
**Pull request by Absolucy**
This is a good ol' holoparasite rebalance. I did some stuff like reducing the costs of several powers (timeskip is nowhere near powerful enough to be worth nearly half your points), reducing the cost of holoparas for traitors (3/4ths of your TC is a bit much for a wildcard that could backfire really bad), made unused holopara injectors refundable, and made holoparas able to stay attached to their owners if they wish (which negates the "pull and punch" strategy, as you'll be pulling the holopara too)

Also, holoparas are now attached to a player's mind rather than body. I imagine this could result in hilarious hijinks with mindswap or podcloning or whatever.

:cl:  Lucy
tweak: Holoparasites are now bound to a user's mind rather than body.
tweak: Holoparasite injectors are now refundable if you don't manage to get a holoparasite.
tweak: Holoparasite injectors now cost 12 TC for traitors, as quite honestly, more often than not they're a wildcard/liability.
tweak: Assassin now costs 3 points.
tweak: The Hand now costs 4 points.
tweak: Limited Healing now costs 2 points.
tweak: Time Erasure now costs 4 points.
fix: Scout holoparasites are now properly invincible when scouting.
tweak: Frenzy's ranged punch now has much higher knockback.
tweak: The Guardian Builder UI is now multi-tabbed, as originally intended.
tweak: There is no more limit to how much you can reset your holoparasites.
tweak: When ghosts are prompted to become a holoparasite, they will be shown the stats and abilities of the holoparasite.
tweak: Holoparasites will be anchored to their owner after manifesting until it chooses to move itself.
/:cl:

### CODE REVIEW

**Comment by KittyNoodle** [comment]
`Holoparasites are now bound to a user's mind rather than body.`
Listen this sounds like a good idea until you remember wizards exist and the statue with a holopara might just be the most horrific thing imaginable

**Comment by Absolucy** [comment]
@KittyNoodle that sounds hilarious

**Comment by Marmio64** [comment]
dont really care about the other stuff but no dusting on death and the holoparasite being tied to the mind instead of the body sounds stupid

**Comment by maxion12345** [comment]
dust on death removal bad
binding to mind sounds based as hell not gonna lie

**Comment by Absolucy** [comment]
Alright, I readded dusting on death

/cc @maxion12345 @Marmio64 @Cartlord @TheLinuxOS

File: code/modules/uplink/uplink_items.dm [hunk 1]
```diff
@@ -446,12 +444,19 @@ GLOBAL_LIST_INIT(uplink_items, subtypesof(/datum/uplink_item))
 	desc = "Though capable of near sorcerous feats via use of hardlight holograms and nanomachines, they require an \
 			organic host as a home base and source of fuel. Holoparasites come in various types and share damage with their host."
 	item = /obj/item/guardiancreator/tech
-	cost = 15
+	cost = 12
 	manufacturer = /datum/corporation/traitor/cybersun
 	surplus = 0
-	//exclude_modes = list(/datum/game_mode/nuclear, /datum/game_mode/nuclear/clown_ops) //yogs: removes restrictions on liabilit- I mean punchghosts
+	exclude_modes = list(/datum/game_mode/nuclear, /datum/game_mode/nuclear/clown_ops)
```
Comment (adamsong) [inline code/modules/uplink/uplink_items.dm]: This change was not mentioned in the PR, why is this being removed from nukies?

File: yogstation/code/modules/guardian/abilities/major/frenzy.dm [hunk 2]
```diff
@@ -29,13 +29,13 @@
 	if(isliving(target) && world.time >= next_rush && guardian.is_deployed())
 		var/mob/living/L = target
 		if(guardian.summoner?.current && get_dist_euclidian(guardian.summoner.current, L) > master_stats.range)
-			to_chat(guardian, "<span class='danger italics'>[L] is out of your range!</span>")
+			to_chat(guardian, span_italics(span_danger("[L] is out of your range!</span>")))
```
Comment (adamsong) [inline yogstation/code/modules/guardian/abilities/major/frenzy.dm]: Closing span tag without opening tag

File: yogstation/code/modules/guardian/abilities/major/frenzy.dm [hunk 3]
```diff
@@ -53,18 +53,18 @@
...
-		to_chat(G, "<span class='danger italics'>[A] is not a living thing.</span>")
+		to_chat(G, span_italics(span_danger("[A] is not a living thing.")))
 		revert_cast()
 		return
 	if(!G.stats)
 		revert_cast()
 		return
 	if(get_dist_euclidian(G.summoner?.current, A) > G.range)
-		to_chat(G, "<span class='danger italics'>[A] is out of your range!</span>")
+		to_chat(G, span_italics(span_danger("[A] is out of your range!</span>")))
```
Comment (adamsong) [inline yogstation/code/modules/guardian/abilities/major/frenzy.dm]: Closing span without an opening

**Review by adamsong — DISMISSED** [review]

File: code/modules/uplink/uplink_items.dm (hunk 1 above)
Comment (Absolucy) [reply code/modules/uplink/uplink_items.dm]: Look below - there's a new subtype exclusive to nukies that is 15 TC. The PR mentions that they cost 12 TC **for traitors**.

Comment (Absolucy) [reply code/modules/uplink/uplink_items.dm]: https://github.com/yogstation13/Yogstation/blob/4f2d846351670abb14890e3e5fd9bd968faf6243/code/modules/uplink/uplink_items.dm#L455-L459

File: code/__DEFINES/components.dm [hunk 4]
```diff
@@ -280,6 +280,7 @@
 #define COMSIG_ITEM_MARK_RETRIEVAL "item_mark_retrieval"			//called before marking an object for retrieval, checked in /obj/effect/proc_holder/spell/targeted/summonitem/cast() : (mob/user)
 #define COMPONENT_BLOCK_MARK_RETRIEVAL 1
 #define COMSIG_ITEM_HIT_REACT "item_hit_react"					//from base of obj/item/hit_reaction(): (list/args)
+#define COMSIG_ITEM_REFUND	"item_refund"						//for tc refunding items: (mob/living/user); returns FALSE if okay to refund, TRUE if not
```
Comment (Bibby0110) [inline code/__DEFINES/components.dm]: feels kinda weird to return TRUE on something NOT okay to refund, and vice-versa

Comment (Absolucy) [reply code/__DEFINES/components.dm]: it's because the results of all signals are OR'd together. 0 | 0 = 0, 0 | 1 = 1, 1 | 1 = 1

Comment (Absolucy) [reply code/__DEFINES/components.dm]: so, something saying "no refund" will take priority

Comment (Bibby0110) [reply code/__DEFINES/components.dm]: Find a solution then.

Comment (Absolucy) [reply code/__DEFINES/components.dm]: Alright, done.

**Review by JamieD1 — APPROVED** [review]

**Comment by KittyNoodle** [comment]
wait so does this work with slimepeople
if it does i foresee a horrifiying new age

**Pull request merged**